packages/rest/src/rest.server.ts

```diff
@@ -151,41 +141,56 @@ export class RestServer extends Context implements Server, HttpServerLike {
    *
    * @param {Application} app The application instance (injected via
    * CoreBindings.APPLICATION_INSTANCE).
-   * @param {RestServerConfig=} options The configuration options (injected via
+   * @param {RestServerConfig=} config The configuration options (injected via
    * RestBindings.CONFIG).
    *
    */
   constructor(
     @inject(CoreBindings.APPLICATION_INSTANCE) app: Application,
-    @inject(RestBindings.CONFIG) options?: RestServerConfig,
+    @inject(RestBindings.CONFIG) config?: RestServerConfig,
   ) {
     super(app);
 
-    options = options || {};
+    config = config || {};
 
     // Can't check falsiness, 0 is a valid port.
-    if (options.port == null) {
-      options.port = 3000;
+    if (config.port == null) {
+      config.port = 3000;
     }
-    if (options.host == null) {
+    if (config.host == null) {
       // Set it to '' so that the http server will listen on all interfaces
-      options.host = undefined;
+      config.host = undefined;
     }
-    this.bind(RestBindings.PORT).to(options.port);
-    this.bind(RestBindings.HOST).to(options.host);
-    this.bind(RestBindings.PROTOCOL).to(options.protocol || 'http');
-    this.bind(RestBindings.HTTPS_OPTIONS).to(options);
 
-    if (options.sequence) {
-      this.sequence(options.sequence);
+    config.openApiSpec = config.openApiSpec || {};
+    config.openApiSpec.endpointMapping =
+      config.openApiSpec.endpointMapping || OPENAPI_SPEC_MAPPING;
+    config.apiExplorer = config.apiExplorer || {};
+
+    config.apiExplorer.url =
+      config.apiExplorer.url || 'https://loopback.io/api-explorer';
+
+    config.apiExplorer.httpUrl =
```

Comment:
Can we come up with a more intuitive interface/names for `apiExplorer.url` and `apiExplorer.httpUrl`?

`httpUrl` makes me wonder if there would be `httpsUrl` as well.

I know it is not trivial and might get overly complicated for a simple thing, but I'd like the ability to set a HTTP url and HTTPS url, and mark one as the default.

Reply:
On second thoughts, I am fine with this. HTTPS is starting to become the expected default in browsers, let HTTPS be the default `apiExplorer.url`.